src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -50,6 +50,12 @@ public class MainViewModel : BaseModel, ISavable, IDisposable
         private bool _saved;
         private ushort _hotkeyHandle;
 
+        private const int _globalHotKeyId = 0x0001;
+        private IntPtr _globalHotKeyHwnd;
```

Comment:
should we initialize those fields to something sensible inline?

Reply:
Default values are 0, so I think there's no need.